lib/jellyfish/room.ex

```diff
@@ -96,6 +96,11 @@ defmodule Jellyfish.Room do
     GenServer.call(registry_id(room_id), {:remove_component, component_id})
   end
 
+  @spec pass_media_event(id(), Peer.id(), String.t()) :: :ok
+  def pass_media_event(room_id, peer_id, event) do
```

Comment:
Nitpick, but I believe that `receive_media_event` is better. Look at https://github.com/cloudflare/quiche#handling-incoming-packets